### ISSUE TITLE
Incorrect "Valid Usage (Implicit)" for xrGetVulkanInstanceExtensionsKHR() and xrGetVulkanDeviceExtensionsKHR()

### ISSUE THREAD

**Issue by puzzlepaint** [KhronosGroup/OpenXR-Docs#77]
For the `xrGetVulkanInstanceExtensionsKHR()` function, the "Parameter Descriptions" documentation says:

```
buffer is a pointer to an array of characters, but can be NULL if bufferCapacityInput is 0. The format of the output is a single space (ASCII 0x20) delimited string of extension names.
```

However, the "Valid Usage (Implicit)" text below says:

```
If bufferCapacityInput is not 0, buffer must be a pointer to an array of bufferCapacityInput null-terminated UTF-8 strings
```

This conflicts with the above. The correct statement is that `buffer` is a pointer to a single string, not to an array of strings.

The same applies to `xrGetVulkanDeviceExtensionsKHR()`, which has a `buffer` parameter with analogous use.

**Comment by rpavlik** [comment]
oh, interesting. I just saw the other side of this issue, over in #69 - this arises from having the `len` attribute of that parameter be `len="bufferCapacityInput,null-terminated"`  - that tells our scripts there is two levels of indirection/array. If I just take off null-terminated, then the generated implicit VU says:

> If bufferCapacityInput is not 0, buffer must be a pointer to an array of bufferCapacityInput char values

Referring to an array of char values feels a little weird, I guess I could special case it in the vu generation script, but that's already the most complicated hairy script since it's like 85% business logic that mostly but not entirely matches between OpenXR and Vulkan...

Does Vulkan have anything that returns a string?

**Comment by rpavlik** [comment]
Oh, apparently we have the same thing `char values` elsewhere in the spec. OK, a change is queued up and will be published in a future release. Thanks for catching this!

**Comment by rpavlik** [comment]
Resolved in 1.0.15. Thanks for your report!
